pkgs/development/compilers/cudatoolkit/redist/build-cuda-redist-package.nix

```diff
@@ -164,11 +164,16 @@ backendStdenv.mkDerivation {
   outputSpecified = true;
 
   meta = {
-    inherit description platforms;
-    license = lib.licenses.unfree;
+    inherit platforms;
+    description = "${description}. By downloading and using the packages you accept the terms and conditions of the ${finalAttrs.meta.license.shortName}";
+    license = lib.licenses.nvidiaCuda // {
+      redistributable = true;
```

Comment:
shouldn't this be under the license definition instead?

Reply:
Do you mean the `// { redistributable = true; }` bit or the "By downloading and using ... " notice?

Reply:
I didn't put `redistributable = true` in the `nvidiaCuda` definition because we also use it for the `cudatoolkit` attribute, which is ambiguous about redistribution (we have to interpret the EULA to say it's redistributable). OTOH the binaries for `cudaPackages.cuda_*` and `cudaPackages.lib*` are fetched from the url which says `redist`, and there are multiple nvidia-adjacent sources which imply the intention to allow redistribution of that stuff.

Back to the code, I could move this into a separate definition, `nvidiaCudaRedist`. I'd still be doing `// { url = "..."; }` because I want the links to lead to package-specific files.

As for the notice, I put it in the package description, because licenses seem to only have two free-form fields: the `shortName`, and `fullName`, and the notice didn't seem to belong in either.

Reply:
The redistributable  part.
Well, if there are two possibilities, then it should be an extra entry on the licenses - something like `nvidiaCudaRedistributable`.